src/main/java/software/sigma/internship/validator/UrlValidator.java

```diff
@@ -0,0 +1,16 @@
+package software.sigma.internship.validator;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class UrlValidator implements ConstraintValidator<Url, String> {
+    @Override
+    public void initialize(Url constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(String s, ConstraintValidatorContext constraintValidatorContext) {
+        return s.matches("[(http(s)?):\\/\\/(www\\.)?a-zA-Z0-9@:%._\\+~#=]" +
```

Comment:
Please move this regex to the static final field as [Pattern](https://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html)